src/bitcoin-cli.cpp

```diff
@@ -51,7 +51,7 @@ static void SetupCliArgs()
     gArgs.AddArg("-rpcport=<port>", strprintf("Connect to JSON-RPC on <port> (default: %u, testnet: %u, regtest: %u)", defaultBaseParams->RPCPort(), testnetBaseParams->RPCPort(), regtestBaseParams->RPCPort()), false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-rpcuser=<user>", "Username for JSON-RPC connections", false, OptionsCategory::OPTIONS);
     gArgs.AddArg("-rpcwait", "Wait for RPC server to start", false, OptionsCategory::OPTIONS);
-    gArgs.AddArg("-rpcwallet=<walletname>", "Send RPC for non-default wallet on RPC server (needs to exactly match corresponding -wallet option passed to bitcoind)", false, OptionsCategory::OPTIONS);
+    gArgs.AddArg("-rpcwallet=<walletname>", "Send RPC for non-default wallet on RPC server (needs to exactly match corresponding -wallet option passed to bitcoind). This changes the RPC endpoint used, e.g. http://127.0.0.1:8332/wallet/<walletname>", false, OptionsCategory::OPTIONS);
```

Comment:
Is this relevant from the `bitcoin-cli` user's point of view? At least I'd drop the `e.g. ...` part here.

Reply:
I'm not sure the best place for this, but I think an example url would be helpful somewhere. There are curl examples for each command help, but not for any of the flags.

Reply:
I disagree with meshcollider and promag, and think this information is useful. Or at least I don't understand how it harms anyone. I hate documentation that just gives me a handwavy gloss about how an option is intended to be used without saying what it actually does.

Of course it would also be good to document wallet endpoints in markdown. There is some text in the [0.15.0 release notes](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.15.0.md#multi-wallet-support) ("HTTP RPC requests should be send to the...") that could be moved to [JSON-RPC-interface.md](https://github.com/bitcoin/bitcoin/blob/master/doc/JSON-RPC-interface.md). This could be done here or in a separate PR. I actually remember complaining about lack of API documentation when this feature was first added https://github.com/bitcoin/bitcoin/pull/10849#issuecomment-315731689, and I guess it still doesn't exist.

Reply:
I generally don't think the option help is the place to put very detailed specifications, but in this case I kind of like it because it's short and directly illustrates what it does.